### ISSUE TITLE
Provide javascript libraries over npm

### ISSUE THREAD

**Issue by netopyr** [canoo/dolphin-platform#189]
Using npm to manage dependencies has some advantages over bower, in particular when using a module bundler like browserify. Thus we should offer the javascript libraries both as bower- and npm-dependencies.

**Comment by hendrikebbers** [comment]
@netopyr but we will still provide DP by bower, right?

**Comment by netopyr** [comment]
Yes, we need to provide both IMO, because sometimes bower fits better and sometimes npm.
